### PR TITLE
Project: Design Teardown by Microverse [Collaborative project] (created with Murilo Roque)

### PR DESCRIPTION
**Pull request by MrSayyod**
- Created a new repository;
- created an extra branch;
- used flex and grid elements;
- set up stickler and linter files;
- committed changes often; 
- HTML file checked by validator.w3.org;
- fixed stickler and stylelint errors.

### CODE REVIEW

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hey  team, 👋

## Good job on this project! 👍

- Good Readme file
- Good Pull Request Message

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Your page shouldn't be an exact clone of the Smashing Magazine website. The purpose of this project is to train your eye to identify the visual hierarchy of a website. That's why you should only include a background color (in a gray scale) for each element in the page and play with some fonts. See Odin requirement:
Screenshot from 2020-03-12 19-41-44

- Please make the necessary changes to achieve the required design. You can take a look a this [page](https://web.archive.org/web/20170628134444/http://www.vanseodesign.com/blog/wp-content/uploads/2009/12/visual-hierarchy-compared.png) to get inspiration.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,539 @@
...
+            </g>
+          </svg>
+        </div>
+        <div>
+          <p>Seriously, red?</p>
+        </div>
+      </div>
+      <div class="vitaly authors">
+        <div class="img-name">
+          <div><img class="persons transformer" src="images/vitaly-friedman.jpg" alt=""></div>
```
Comment (majovanilla) [inline index.html]: There is no need to include images in your project

File: index.html [hunk 2]
```diff
@@ -0,0 +1,539 @@
...
+            <li>#Privacy <sup>13</sup></li>
+            <li>#UX <sup>1</sup></li>
+            <li>#Smashing books <sup>37</sup></li>
+          </ul>
+          <p class="arrow">→</p>
+        </div>
+      </div>
+    </div>
+  </main>
+  <article class="latest-posts">
```
Comment (majovanilla) [inline index.html:165]: The latest post section it's different from the page. The first row should have 3 articles of the same size but the next two have one a little larger.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+        <h1>New on smashing TV</h1>
+        <p>A channel where seasoned developers show how they design, build, work and play. Coming up next: <a
+            href="#">Building A Modern Electron App With React Hooks</a> with Cassidy Williams (Mar 19).</p>
+        <h4> See all upcoming episodes →</h4>
+        <h5>Already have an account? <a href="#">Sign in!</a></h5>
+      </div>
+    </div>
+    <p class="see-all">See all latest articles →</p>
+  </article>
+  <section class="ethical">
```
Comment (majovanilla) [inline index.html:235]: This section doesn't need to include the images and bg color. That way you can highlight it using a background color that reflects its importance.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,539 @@
...
+      <h5>March 2020 • Shipping now</h5>
+      <h1><a href=""> Ethical Design Handbook:</a></h1>
+      <h2>Leaving Dark Patterns Behind.</h2>
+      <p>With <strong> practical techniques & strategies</strong> to make honest design work for digital products.
+        Help your business grow sustainably — without dark patterns — and comply with GDPR and CCPA.</p>
+      <h4>Learn more about the book →</h4>
+      <img src="images/ethical-design-transparent.png" alt="Ethical Deisgn">
+    </div>
+  </section>
+  <section class="community">
```
Comment (majovanilla) [inline index.html:246]: Get rid of the image and the text, just highlight the elements inside this section

File: index.html [hunk 5]
```diff
@@ -0,0 +1,539 @@
...
+              raise awareness for creating truly inclusive websites, he <a href="#"> designed tailor-made sites for and
+                with
+                people with real disabilities </a> for his thesis. He shares his knowledge about web design and
+              development on his <a href="#"> blog</a>, in workshops, at conferences, and giving lectures at schools and
+              meetups.
+              Thank you for everything you do for the community, dear Vasilis!</p>
+          </div>
+        </div>
+      </div>
+      <div class="grid-part">
```
Comment (majovanilla) [inline index.html:268]: The same here, no need to add the image, just the div that contains it and select a color based on its visual hierarchy.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,539 @@
...
+              require several files with a single file and some CSS. The <a href="#"> “A Variable Fonts Primer”</a>
+              teaches you how variable fonts tick and how they can benefit UI design, accessibility, and long-form
+              reading.</p>
+          </div>
+        </div>
+        <a href="#">More links on Twitter → </a>
+      </div>
+    </div>
+  </section>
+  <section class="featured">
```
Comment (majovanilla) [inline index.html:354]: This shouldn't have a red bg color, just the correspondent gray.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,539 @@
...
+      <h3>Featured Smash Book</h3>
+      <h1><a href="#"> Design Systems</a></h1>
+      <p>Meet <strong> Design Systems</strong>, a recent Smashing book in which Alla Kholmatova explores how to set up
+        an <strong> effective
+          design system</strong> to create great digital products. With common traps, gotchas and lessons learned.</p>
+      <h4>Table of contents ↬</h4>
+    </div>
+    <img class="transformer" src="images/design-systems-hardcover-book.png" alt="">
+  </section>
+  <section class="grid-part2">
```
Comment (majovanilla) [inline index.html:365]: You should only add text to a selected area of your page. The rest can be just divs.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,539 @@
...
+          <h5>Your (smashing) email</h5>
+          <h3><a href="#">Subscribe →</a></h3>
+        </div>
+        <h6><a href="#">Useful tips for web designers.</a> Sent 2× a month.
+          You can unsubscribe any time — <em>obviously.</em></h6>
+      </div>
+    </div>
+    <p class="see-all2">See all Articles →</p>
+  </section>
+  <footer>
```
Comment (majovanilla) [inline index.html:438]: You don't have to delete the text of the footer but you do need to change the background color to reflect the visual importance of the section.

**Review by majovanilla — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (MuriloRoque) [reply index.html]: Done!

File: index.html (hunk 2 above)
Comment (MuriloRoque) [reply index.html:165]: Done!

File: index.html (hunk 3 above)
Comment (MuriloRoque) [reply index.html:235]: Done!

Comment (MrSayyod) [reply index.html:235]: Done

File: index.html (hunk 6 above)
Comment (MrSayyod) [reply index.html:354]: Done

File: index.html (hunk 5 above)
Comment (MrSayyod) [reply index.html:268]: Done

File: index.html (hunk 4 above)
Comment (MrSayyod) [reply index.html:246]: Done

Comment (MuriloRoque) [reply index.html:246]: Done!

File: index.html (hunk 5 above)
Comment (MuriloRoque) [reply index.html:268]: Done!

File: index.html (hunk 6 above)
Comment (MuriloRoque) [reply index.html:354]: Done!

File: index.html (hunk 7 above)
Comment (MuriloRoque) [reply index.html:365]: Done!

File: index.html (hunk 8 above)
Comment (MuriloRoque) [reply index.html:438]: Done!

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Great work team!
You have created a perfect clone of the website, however, this isn't the aim for this project as my colleague pointed out in your last review. Please take a look at the example below to see what is expected of you and make the changes before your next review. I wish you the best and see at the top

#### Design Teardown Example
[Example](https://rawcdn.githack.com/carlosprietobarron/smashingmag/2af4d72ed7e3f7e857d8bdc8e2b8637ec1a6bf33/index.html)

##### Think I missed something? Need some help with this or any of your projects? Please feel free to reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilare_olusakin), even if you just want to chat and I will respond to you as soon as I can.

**Comment by MrSayyod** [comment]
Thanks for the review! WE have already done the changes, but we didn't update the README.md file. Now we updated it and you can check it out again.

**Comment by lekegitrepo** [comment]
### Hi Team 👏 👏 💯 🥇 
### This is good, you have done exceptionally well by applying the required changes. This project is approved.

### Reviewed by: Temitayo Adeleke

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
### Status: Approved.

**Pull request merged**